x-pack/plugin/ilm/src/main/java/org/elasticsearch/xpack/ilm/IlmHealthIndicatorService.java

```diff
@@ -103,62 +131,63 @@ public class IlmHealthIndicatorService implements HealthIndicatorService {
         )
     );
 
-    private static final TimeValue ONE_DAY = TimeValue.timeValueDays(1);
-    private static final long MAX_RETRIES = 100;
-
-    static final Map<String, RuleConfig> RULES_BY_ACTION_CONFIG = Map.of(
+    static final Map<String, RuleCreator> RULES_BY_ACTION_CONFIG = Map.of(
```

Comment:
I won't put it there because it's like _the default set of rules_ that will be passed to `StagnatingIndicesFinder`... It could live there or in IlmHealthIndicator or in its own file.